web/src/components/studies/CreateStudyForm.tsx

```diff
@@ -401,8 +432,18 @@ export default function CreateStudyForm(CreateStudyProps: CreateStudyProps) {
                       value: 999,
                       message: "Cannot be more than 3 digits",
                     },
-                  })}
-                  placeholder="eg 123"
+                  }}
+                  render={({ field }) => (
+                    <input
+                      {...field}
+                      type="number"
+                      id="dataProtectionId"
+                      placeholder="eg 123"
+                      style={{ padding: "0.5rem", fontSize: "1rem", border: "1px solid #ccc", borderRadius: "4px" }}
```

Comment:
if we're rounding the outer edges, we probably also want to do this for the registry ID too

Reply:
I think UCL style is no rounding – so we should be consistent and have none anywhere(?)

Reply:
https://github.com/ucl-arc-tre/portal/pull/172/commits/9d33c7fbfbaca2a32c7bb6c4e29dd440d9fa15b6

I removed the border radius and inline styles 👍